app/models/kit_allocation.rb

```diff
@@ -13,5 +13,4 @@
 class KitAllocation < ApplicationRecord
   include Itemizable
   belongs_to :storage_location
-  enum kit_allocation_type: {inventory_in: 0, inventory_out: 1}
```

Comment:
Why was this renamed from `kit_allocation_type`? I feel like that was a more indicative name.

Reply:
reused kit_allocation_type, the reason to rename to "inventory" was conventional way to give name to enum_type: is "model_name_enum_name"

ie 
`# db/migrate/20131220144913_create_articles.rb
def up
  create_enum :article_status, ["draft", "published"]

  create_table :articles do |t|
    t.enum :status, enum_type: :article_status, default: "draft", null: false
  end
end
`

so it was becoming "kit_allocation_kit_allocation_type". now  just used "kit_allocation_type" bypassing that convention